BDArmory/Competition/BDACompetitionMode.cs

```diff
@@ -209,6 +209,26 @@ void OnGUI()
                         {
                             guiStatusString += $"\nCurrent Firing Rate: {BDArmorySettings.FIRE_RATE_OVERRIDE} shots/min.";
                         }
+                        if (BDArmorySettings.RUNWAY_PROJECT_ROUND == 67 && pinataAlive && BDArmorySetup.GAME_UI_ENABLED && !MapView.MapIsEnabled)
```

Comment:
This should also have a `BDArmorySettings.RUNWAY_PROJECT` condition. It'd probably be better to make it into a common check to include the round 41 check above this.